pypsa/plot.py

```diff
@@ -56,6 +56,7 @@
     import cartopy
     import cartopy.crs as ccrs
     import cartopy.mpl.geoaxes
+    import requests
```

Comment:
`import requests` to fix [this line](https://github.com/PyPSA/PyPSA/blob/0b7d01c5144db28cffe97b0519492487d8bdb6fe/pypsa/plot.py#L287).